<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InputForm</class>
 <widget class="QDialog" name="InputForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>90</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Input form</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
   <property name="inputMask">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonAdd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>53</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonSubstract">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>53</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonMultiply">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>53</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>*</string>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDivide">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>54</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>/</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
